tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml

```diff
@@ -177,4 +177,119 @@ TestOnboardersAccessMyAcountWindowAndReviewMyInfo:
      Value: OnboarderCarePartnerLogin
   # Call case to access "My Account" Window and Review "My Info" Page.
    - Type: case
-     Value: CarePlatformUpdateInfo
\ No newline at end of file
+     Value: CarePlatformUpdateInfo
+
+# CXTB-143: Access Participant Directory & Filter
+TestOnboarderAccessParticipantDirectoryandFilter:
+    Environment: Settings 
+    Roles:
+    - Role: desktopChrome
+      App: "desktop"
+    Actions:
+  # Log in Care Platform as an Onboarder
+    - Type: case
+      Value: OnboarderCarePartnerLogin
+  # Go to Participants
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+  # Get the value to compare later.
+    - Type: get_text
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_darkened_letter_button$
```

Comment:
if this object represents the last name of a participant, it should be included in the name, first darkened letter button does not suggest what it is. When we create an object, we do it with the intention for it to be reused by our peers.

Reply:
I can see you were not the one to add this object, so is up to you if you want to correct that.

Reply:
Its ok, I will update the object.